operator/operator.go

```diff
@@ -28,6 +28,7 @@ type Operator struct {
 	EthClient *eth.Client
 	*depositProofs
 	*balanceProofs
+	*TxReceipts
```

Comment:
yes, needs to be public because the operator test needs to access it. I was just following the pattern used for the deposit proofs and balance proofs.

Reply:
OK... I think none of them should be embedded but just fields instead. Maybe at least make this a field, can have the same name as the type.